CHANGELOG.md

```diff
@@ -4,11 +4,15 @@
 
 Enhancements:
 - .NET Standard 2.0 and 2.1 support (@lg2de, #485)
+- Non-intercepted methods on a class proxy with target are now forwarded to the target (@stakx, #571)
 
 Bugfixes:
+- Not proxied methods on class proxy with target should forward to the target (DYNPROXY-183) (@jonorossi, #67)
 - Proxying certain `[Serializable]` classes produces proxy types that fail PEVerify test (@stakx, #367)
 - `private protected` methods are not intercepted (@CrispyDrone, #535)
+- Virtual property is not proxied if `IProxyGenerationHook.ShouldInterceptMethod` returns false (@xumix, #536)
```

Comment:
We probably don't need three separate entries, but I wasn't sure about the changelog entry format for this situation.

Reply:
Yer, definitely don't need all 3. I'd say this is an enhancement not a defect at this stage, and just keep your issue number, my name and the last one shouldn't be mentioned, neither of us contributed the change.

Reply:
OK, will do. I'd still want to have a trace back to the original issues, but I suppose it might be enough if they're in the PR.